etc/LATISS/primary_hdu.header

```diff
@@ -40,11 +40,11 @@ RAEND   =                      / RA of telescope from AZEND and ELEND
 DECEND  =                      / DEC of telescope from AZEND and ELEND
 ROTPA   =                      / Rotation angle relative to the sky (deg)
 ROTCOORD= 'sky'                / Telescope Rotation Coordinates
-HASTART =                      / [HH:MM:SS] Telescope hour angle at start
+HASTART =                      / Telescope hour angle at start [decimal hrs]
```

Comment:
Units should go at the start of the string in FITS convention (which means that ROTPA is wrong above but ELEND is correct below).

in this case though "decimal hours" is not an official fits units string so it would be best to leave it at the end so tooling doesn't get confused.

The standard says that you should use `[h]` so can you please switch to that?

Reply:
Thanks for catching this. It's been updated.

Reply:
@timj Did you mean change to `[decimal h]` or just `[h]`

Reply:
That sounds good to me.

Reply:
When I said `[h]` I meant `[h]`.